__fixtures__/test-project/package.json

```diff
@@ -20,8 +20,5 @@
   "prisma": {
     "seed": "yarn rw exec seed"
   },
-  "packageManager": "yarn@3.6.3",
-  "resolutions": {
-    "@types/react": "18.2.14"
-  }
+  "packageManager": "yarn@3.6.3"
```

Comment:
Think this should go back :).

Not sure what we need it for though.

Reply:
Just merge main into this branch and this change should go away